quipucords/api/source/tests_source.py

```diff
@@ -1116,6 +1181,27 @@ def test_update_sat_more_than_one_hosts(self):
                                    format='json')
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
+    def test_update_sat_more_exclude_hosts(self):
+        """Sat- Fail update due to multiple hosts."""
```

Comment:
This docstring should be like the vcenter one : """Sat - Fail update due to exclude hosts."""